app/src/main/java/org/mozilla/fenix/search/awesomebar/AwesomeBarView.kt

```diff
@@ -111,7 +111,7 @@ class AwesomeBarView(
                 selectTabUseCase,
                 components.core.icons,
                 getDrawable(activity, R.drawable.ic_search_results_tab),
-                excludeSelectedSession = true
+                excludeSelectedSession = false
```

Comment:
Thanks for taking a look at this @mawen7 !
I'm afraid setting `excludeSelectedSession` to `false` would reintroduce the https://github.com/mozilla-mobile/android-components/issues/3578 issue. Basically if you had one open tab (say wikipedia.org) and you would start typing in the url bar "wikipedia" the switch-to-tab option would appear for the tab you were on.

To fix the issue we would need to somehow detect if the user is on the homeFragment when passing  `excludeSelectedSession`

Reply:
Hey @codrut-topliceanu thanks for your review.
I can confirm your described behavior shows up for me except it now just switches to the current tab instead of doing nothing.

Based on [@ValentinTimisica comment](https://github.com/mozilla-mobile/fenix/issues/9457#issuecomment-605028558) and [@brampitoyo answer](https://github.com/mozilla-mobile/fenix/issues/9457#issuecomment-671747472):
> 
> 
> We definitely want to show (and visually label) currently open tabs in address bar search results (if it matches query), and allow people to easily switch to it.

I thought it is the new intended behavior.

Maybe UX can confirm? Taking into consideration mozilla-mobile/android-components#3578 isn't as bad as it was.
Otherwise I can take another look how to include current session only if opening search from `HomeFragment`.

Reply:
@codrut-topliceanu @mawen7 TL;DR – I think `excludeSelectedSession` should be set to `true`.

I should have clarified in my comments earlier.

Open tabs are shown in address bar search results, but _not_ the same tab that you are currently on.

For example, I have 4 open tabs:

- wikipedia.org
- wiki.mozilla.org
- commons.wikimedia.org
- New Tab Page

My current tab is _New Tab Page_. I then select the address bar and enter the keyword “wiki”. The search results should display:

- wikipedia.org
- wiki.mozilla.org
- commons.wikimedia.org

I am now switching tab to _wiki.mozilla.org_, then enter the same keyword into the address bar. The results would display:

- wikipedia.org
- commons.wikimedia.org

It does not display wiki.mozilla.org, because it’s the very same tab I’m currently on.

Hope this makes the behaviour clear?

Reply:
Thank you for making this clear and the detailed example!